.prettierignore

```diff
@@ -11,6 +11,7 @@ app/node_modules/
 
 app/static/common
 app/test/fixtures
+app/node_modules
```

Comment:
Is this needed? Prettier should ignore `node_modules` by default, and if it doesn’t then that’s a bug.

Reply:
oh! didn't realize that